lib/extensions/ar_migration.rb

```diff
@@ -52,11 +28,41 @@ def self.update_local_migrations_ran(version, direction)
         WHERE id = #{region.id}
       SQL
     end
+
+    class RemoteRegionThing
```

Comment:
You are the master of names LOL

Reply:
:laughing: I changed it like 4 times before giving up and using this just so I could continue re-working the code.